src/ui/public/vis/vis_types/vislib_vis_legend.html

```diff
@@ -3,12 +3,14 @@
     type="button"
     ng-click="toggleLegend()"
     class="kuiCollapseButton visLegend__toggle"
+    ng-class="{'visLegend__toggle--isOpen': open}"
     aria-label="{{::'common.ui.vis.visTypes.legend.toggleLegendButtonAriaLabel' | i18n: { defaultMessage: 'Toggle legend' } }}"
     aria-expanded="{{!!open}}"
     aria-controls="{{::legendId}}"
     data-test-subj="vislibToggleLegend"
+    title="{{::'common.ui.vis.visTypes.legend.toggleLegendButtonAriaLabel' | i18n: { defaultMessage: 'Toggle legend' } }}"
```

Comment:
Even if it's the same string on the same element, the way the i18n was done so far, we should use different ids, so rather `common.ui.vis.visTypes.legend.toggleLegendButtonTitle` for this string.